transform/logs/dbt.log

```diff
@@ -0,0 +1,53 @@
+[0m14:27:21.112735 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10274add0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1042c4a90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1042cbed0>]}
```

Comment:
The .gitignore file has the logs/ directory entry in it. The issue is that the logs are one level above, so it can't find it. I will fix the gitignore path to logs/ . The gitignore file looks the same as the one in dlt core.